Flow.Launcher.Plugin/SharedCommands/SearchWeb.cs

```diff
@@ -1,18 +1,45 @@
+using Microsoft.Win32;
 using System;
 using System.Diagnostics;
 using System.IO;
 using System.Linq;
 
+
 namespace Flow.Launcher.Plugin.SharedCommands
 {
     public static class SearchWeb
     {
+        private static string GetDefaultBrowserPath()
+        {
+            string name = string.Empty;
+            try
+            {
+                var regDefault = Registry.CurrentUser.OpenSubKey("Software\\Microsoft\\Windows\\CurrentVersion\\Explorer\\FileExts\\.htm\\UserChoice", false);
```

Comment:
why
`Software\\Microsoft\\Windows\\CurrentVersion\\Explorer\\FileExts\\.htm\\UserChoice`
and not
`Software\\Microsoft\\Windows\\Shell\\Associations\\UrlAssociations\\http\\UserChoice`
?

Reply:
pleas also add a `using` here

Reply:
OK, I am not sure about the use of registry since I am not familier with it, so I just copy one and it seems it works. I will test the one you suggested and it seems it will be more reasonable to use that one instead of the htm default link.